app/Http/Controllers/CampaignsController.php

```diff
@@ -33,6 +33,18 @@ public function index(Request $request)
         $filters = $request->query('filter');
         $query = $this->filter($query, $filters, Campaign::$indexes);
 
+        // Apply scope for the "computed" is_open field:
+        if (isset($filters['is_open'])) {
+            // We can use this to find only open campaigns with a truthy filter
+            // applied, e.g. `?filter[is_open]=true`, or closed campaigns with
+            // a falsy filter, e.g. `?filter[is_open]=false`.
```

Comment:
Curious if helpful at all to just have a corresponding `is_closed` filter, just so it's available if preferred to read it as such? Defs not a blocker and not needed for now, but could be a nice to have 🤷‍♂

Reply:
Yeah, that's an interesting idea. 📕